lms/djangoapps/teams/static/teams/js/spec/views/teams_spec.js

```diff
@@ -109,45 +120,77 @@ define([
+                teamMembership = createTeamMembership(teamMembershipData),
+                teamsView = createTeamsView({
+                    teams: teamMembership,
+                    teamMembership: teamMembership
+                });
 
             expect(teamsView.$('.teams-paging-header').text()).toMatch('Showing 1-5 out of 11 total');
             var footerEl = teamsView.$('.teams-paging-footer');
             expect(footerEl.text()).toMatch('1\\s+out of\\s+\/\\s+3');
             expect(footerEl).not.toHaveClass('hidden');
 
-            verifyCards(teamsView, initialTeamMemberships);
+            verifyCards(teamsView, teamMembershipData);
         });
 
-        it ('can render the actions view', function () {
-            teamsView = new TeamsView({
-                el: '.teams-container',
-                collection: teamCollection,
-                teamParams: {},
-            }).render();
+        describe("Team Actions View", function() {
+            it('can render itself correctly', function () {
+                var emptyMembership = createTeamMembership([]),
+                    teamsView = createTeamsView({ teamMembership: emptyMembership });
+                expect(teamsView.$('.title').text()).toBe('Are you having trouble finding a team to join?');
+                expect(teamsView.$('.copy').text()).toBe(
+                    "Try browsing all teams or searching team descriptions. If you " +
+                    "still can't find a team to join, create a new team in this topic."
+                );
+            });
 
-            expect(teamsView.$el.text()).not.toContain(
-                'Are you having trouble finding a team to join?'
-            );
 
-            teamsView = new TeamsView({
-                el: '.teams-container',
-                collection: teamCollection,
-                teamParams: {},
-                showActions: true
-            }).render();
+            it('can navigate to correct routes', function () {
+                var emptyMembership = createTeamMembership([]),
+                    teamsView = createTeamsView({ teamMembership: emptyMembership });
+                spyOn(Backbone.history, 'navigate');
+                teamsView.$('a.browse-teams').click();
+                expect(Backbone.history.navigate.calls[0].args).toContain('browse');
 
-            expect(teamsView.$el.text()).toContain(
-                'Are you having trouble finding a team to join?'
-            );
+                teamsView.$('a.search-team-descriptions').click();
+                // TODO! Should be updated once team description search feature is available
+                expect(Backbone.history.navigate.calls[1].args).toContain('browse');
+
+                teamsView.$('a.create-team').click();
+                expect(Backbone.history.navigate.calls[2].args).toContain('topics/test-topic/create-team');
+            });
+
+            it('shows for a user not in a team', function () {
```

Comment:
Good point. When I collapsed team_actions_spec into this file I didn't notice that. I'll remove this method.